### DISCUSSION TITLE
Label-based animation export

### DISCUSSION THREAD

**Discussion by farfalk** [Orama-Interactive/Pixelorama#472] [Ideas]
I'm using 0.8.3-dev, and it's currently possible to label frames and play animations using only the frames that share a specific label. I think it would be useful to optionally limit the animation export to selected labels as well.

**Comment by novhack** [comment]
For sure. I actually planned it for v0.9 but kinda forgot. I'm writing this down for future export improvements.

**Comment by OverloadedOrama** [comment]
Specific frame tag exporting now works for all exporting types (multiple pngs, spritesheets, gif/apng) thanks to #781 and it will be included in v0.11.
